src/reactive/easyrsa.py

```diff
@@ -109,23 +109,31 @@ def configure_easyrsa():
 
 
 def configure_copy_extensions():
-    """Update the EasyRSA configuration with the capacity to copy the exensions
+    """Update the EasyRSA configuration with the capacity to copy the extensions
     through to the resulting certificates."""
-    # Create an absolute path to the file which will not be impacted by cwd.
-    (openssl_file,) = glob.iglob(os.path.join(easyrsa_directory, "openssl-*.cnf"))
+
+    openssl_confs = Path(easyrsa_directory).glob("openssl-*.cnf")
```

Comment:
in 3.0.1 -- the charm uses a file `EasyRSA-3.0.1/openssl-1.0.cnf`
in 3.0.9 -- the charm uses a file `EasyRSA-v3.0.9/openssl-easyrsa.cnf` which is copied to `EasyRSA-v3.0.9/pki/openssl-easyrsa.cnf`  if the `pki` directory exists

on an upgrade, the `pki` directory is copied from `EasyRSA-3.0.1/pki/` to `EasyRSA-v3.0.9/pki`